iothub_client/samples/iothub_client_device_twin_and_methods_sample/iothub_client_device_twin_and_methods_sample.c

```diff
@@ -217,6 +217,13 @@ static int deviceMethodCallback(const char* method_name, const unsigned char* pa
     return result;
 }
 
+static void getCompleteDeviceTwinOnDemandCallback(DEVICE_TWIN_UPDATE_STATE update_state, const unsigned char* payLoad, size_t size, void* userContextCallback)
+{
+    (void)update_state;
+    (void)userContextCallback;
+    printf("GetDeviceTwinAsync result:\r\n%.*s\r\n", (int)size, payLoad);
```

Comment:
>GetDeviceTwinAsync [](start = 12, length = 18)

Shouldn't this be GetCompleteTwin, just to be explicit.

Reply:
I started a conversation with the team about it.

Reply:
The team has decided to keep the implementation as currently done.